en/collect/import-using-online-bibliographic-database.md

```diff
@@ -129,6 +129,10 @@ The following list shows some of the field indicators that can be used:
 
 [Jstor](https://jstor.org) is an online database with access to more than 12 million journal articles, books and sources in 75 disciplines. [About](https://about.jstor.org/)
 
+To fetch entries from Jstor the corresponding fetcher has to be selected from the drop-down menu over the WebSearchBar by scrolling downwards:
```

Comment:
I'm against adding this for each fetcher. This would produce a long list of images whose only change is the selected drop-down image. You could add a slightly reformulated version at the end of this list here, showing more generally how one can select the different fetchers.

Reply:
Yeah, I agree with that. I just added a notice to the overall search description. Do you think this is sufficient or will an image be necessary? :)

Reply:
One image at the beginning is great. That was the point why the request for the documentation has been risen. See https://github.com/JabRef/jabref/issues/6627#issuecomment-733132434 and the next comment https://github.com/JabRef/jabref/issues/6627#issuecomment-733134858 for details.

Reply:
I actually don't think a screenshot is really necessary, a short sentence describing that one can select the database from the dropdown menu would be sufficient in my opinion.

The image shown in
https://github.com/JabRef/user-documentation/blob/6ee4b1e6daa69b859fa796669b325674439d1b01/en/collect/import-using-online-bibliographic-database.md
is definitely to prominent. Can you please reduce the size.

Reply:
I am thinking of persons [more addicted to pictures](https://lernumgebungen.wordpress.com/der-visuelle-lerntyp/) than [texts](https://lernumgebungen.wordpress.com/der-lesen-und-schreiben-lerntyp-oder-der-skriptoriker/) ("Skriptoriker"). I want to cover both target groups. Thus, we should keep a picture.

I agree that it too small. I think, we can use the html `<img>` tag here. See https://stackoverflow.com/a/11117687/873282 for details.